tools/c7n_left/c7n_left/cli.py

```diff
@@ -0,0 +1,48 @@
+# Copyright The Cloud Custodian Authors.
+# SPDX-License-Identifier: Apache-2.0
+#
+import logging
+from pathlib import Path
+
+import click
+from c7n.config import Config
+
+from .output import get_reporter, report_outputs
+from .provider import CollectionRunner
+from .utils import load_policies
+
+
+@click.group()
+def cli():
+    """Shift Left Policy"""
+    logging.basicConfig(level=logging.DEBUG)
+
+
+@cli.command()
+@click.option("--format", default="terraform")
+@click.option("-p", "--policy-dir", type=click.Path())
+@click.option("-d", "--directory", type=click.Path())
```

Comment:
From the bits of initial testing, I'm wondering if this might be more usable as either a path pattern/glob or paired with a `recursive` option to descend into child directories

Reply:
yeah.. I'd like to save that for a future capability (scan arbitrary repo), for now the intent is point to a root module dir.